torch/csrc/jit/serialization/export.cpp

```diff
@@ -708,7 +708,10 @@ void EncoderBase::AddAttribute(
       }
       break;
     default:
-      throw std::runtime_error("unexpected attribute kind");
+      std::ostringstream err_msg;
+      err_msg << "attribute \"" << name.toDisplayString()
+              << "\" has unexpected kind: " << toString(node->kindOf(name));
```

Comment:
Sorry, this file shouldn't have been included in this PR at all. Removed.
(for what it's worth I added this because I hit this error and this change worked as expected to produce a more detailed error message)